src/index.js

```diff
@@ -126,6 +128,10 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
     render() {
       let { pastDelay, error, Component } = this.state;
 
+      if (!isWebpack && serverSideRequirePath) {
```

Comment:
@phyllisstein would you be so kind to publish your full solution? Can't make it work, because flushServerSideRequirePaths always returns path to the main bundle, and not paths of the loaded chunks.